src/main/main.ts

```diff
@@ -38,29 +39,27 @@ if (
 
 require('@electron/remote/main').initialize();
 
-// const installExtensions = async () => {
-//   const installer = require('electron-devtools-installer');
-//   const forceDownload = !!process.env.UPGRADE_EXTENSIONS;
-//   const extensions = ['REACT_DEVELOPER_TOOLS'];
-//
-//   return installer
-//     .default(
-//       extensions.map((name) => installer[name]),
-//       forceDownload
-//     )
-//     .catch(console.log);
-// };
+interface BrowserViewsMap {
+  [key: number]: BrowserView;
+}
 
 const windows = new Set();
+let menuBuilder: MenuBuilder | undefined;
+const browserViews: BrowserViewsMap = {};
+let lastTopBrowserView: BrowserView | null = null;
+
+function refreshViewBounds(_win?: BrowserWindow, _view?: BrowserView) {
+  if (!_win || !_view) {
+    return;
+  }
 
-function refreshViewBounds(_win, _view) {
-  const bounds = _win.getBounds();
-  const viewBounds = _view.getBounds();
+  const bounds = _win?.getBounds();
+  const viewBounds = _view?.getBounds();
 
   let offset = 0;
   if (os.platform() === 'win32') {
     // eslint-disable-next-line no-nested-ternary
-    offset = _win.isMaximized()
+    offset = _win?.isMaximized()
```

Comment:
Seems like _win should already be defined, since you have a check in the beginning of this function, so no need for optional chaining